_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+---
+title: 'Bitcoin Optech Newsletter #223'
+permalink: /en/newsletters/2022/10/26/
+name: 2022-10-26-newsletter
+slug: 2022-10-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Continued discussion about full RBF:** in [last week's
+  newsletter][news222 rbf], we summarized a discussion on the
+  Bitcoin-Dev mailing list about the inclusion of a new `mempoolfullrbf`
+  option that could create problems for a several businesses which
+  accept transactions with zero confirmations ("zeroconf") as final
+  payments.  Discussion continued this week on both the mailing list and
+  the #bitcoin-core-dev IRC room.  Some highlights of the discussion
+  include:
+
+    - *Free option problem:* Sergej Kotliar [warned][kotliar free
+      option] that he believes the greatest problem with transaction
+      replacement is that it creates a free American call option.  For
```

Comment:
My sincerest apologies if I seem argumentative here, but I interpreted his message differently. To me, it seemed that he was describing an existing problem that is worsened with easier replacements - not to dismiss this point, just that "create" doesn't seem like the right word.
```suggestion
      option] that he believes the greatest problem with transaction
      replacement is that it eases a free American call option.  For
```
![image](https://user-images.githubusercontent.com/25183001/197531516-dbdb984f-ecd2-4b58-a1e0-4eea702afc12.png)

Reply:
First, please argue!  Second, I phrased this section around "replacements" in general, rather than "full RBF" in particular, in order to try to convey the point that it's any type of replacement that *creates* the problem.  I'll edit to see if I can make that clearer.

Reply:
Ah, I see I didn't catch that this was about replacements in general!